pkg/dockerfile/fast_generator.go

```diff
@@ -288,10 +288,12 @@ func (g *FastGenerator) installPython(lines []string, tmpDir string) ([]string,
 		return nil, err
 	}
 	if requirementsFile != "" {
+		srcMount := "--mount=type=bind,ro,source=.,target=/src"
```

Comment:
Python wheel/zip files in `requirements.txt` cannot be in `.dockerignore` or they'll not be exposed via this bind mount. OTOH they'll be baked into the `/src` layer unless we parse requirements, check existence and add them to `--exclude` in the `COPY` step, overkill for now assuming most private packages are small.